electricitymap/contrib/capacity_parsers/tests/test_REE.py

```diff
@@ -0,0 +1,120 @@
+from datetime import datetime
+
+from electricitymap.contrib.capacity_parsers import REE
+
+
+class FakeResponse:
+    def __init__(self, status_code: int, payload: dict):
+        self.status_code = status_code
+        self._payload = payload
+
+    def json(self):
+        return self._payload
+
+
+class FakeSession:
+    def __init__(self, response: FakeResponse):
+        self._response = response
+        self.last_call = None
+
+    def get(self, url, params=None):
+        self.last_call = (url, params)
+        return self._response
+
+
+def _patch_minimal_mappings(monkeypatch):
+    """
+    Para tornar o teste independente do conteúdo real de REE.py,
+    reduzimos os dicionários a um caso mínimo estável.
+    """
+    monkeypatch.setattr(REE, "ZONE_KEY_TO_GEO_LIMIT", {"ES": "system"}, raising=False)
+    monkeypatch.setattr(REE, "GEO_LIMIT_TO_GEO_IDS", {"system": "es"}, raising=False)
+    # Mapeamento mínimo de tipos → modes
+    monkeypatch.setattr(
+        REE,
+        "MODE_MAPPING",
+        {
+            "solar": "solar",
+            "hydro": "hydro",
+        },
+        raising=False,
+    )
```

Comment:
Why is this mocked?

Also please use english so other community members and our employees can understand the comments (including me).